pkgs/development/python-modules/stim/default.nix

```diff
@@ -32,7 +32,7 @@ buildPythonPackage rec {
   postPatch = ''
     # asked to relax this in https://github.com/quantumlib/Stim/issues/623
     substituteInPlace pyproject.toml \
-      --replace "pybind11==" "pybind11>="
+      --replace "pybind11~=" "pybind11>="
```

Comment:
```suggestion
      --replace-warn "pybind11~=" "pybind11>="
```

Reply:
Or use  `--replace-fail`if strict replace is required

Reply:
eh, it doesn't really matter. the failure case if the substitution fails is the build is fine because the versions match or the build fails because they don't. and the failure case already logs and is plenty clear:

https://hydra.nixos.org/build/257538603/nixlog/1/tail

so i went with quiet.